extras/claude.sh

```
@@ -0,0 +1,156 @@
+"I want you to work on GitHub shader-slang/'$githubRepo' issue $githubIssue.
+
+Phase: Implementation
+
+1. Read plan.best.md carefully — this is your implementation blueprint.
+2. Re-read the issue and any linked PRs to confirm your understanding before touching code.
+3. Implement the solution exactly as specified in plan.best.md.
+4. Do NOT commit changes yet." | tail -1)
+
+log "Phase 4: Refining the implementation..."
+
+refine_session=$(claude_json --resume "$impl_session" -- \
```

Shellcheck (0.11.0)

[warning] refine_session appears unused. Verify use (or export if used externally).

(SC2034)